webgallery/static/css/webgallery.css

```diff
@@ -15,11 +15,11 @@ li {
    list-style: none;
 }
 
-.slider {
+.big-pcture {
```

Comment:
non descriptive name, typo, prefix your gallery style with some name like aliceGallery-... to avoid potentiall name collisions in css

Reply:
added prefix aw-gallery- to all css selectors connected with the gallery.